projects/dsp-ui/src/lib/action/directives/external-links/external-links.directive.ts

```diff
@@ -0,0 +1,39 @@
+import { isPlatformBrowser } from '@angular/common';
+import { Directive, HostBinding, Inject, Input, OnChanges, PLATFORM_ID } from '@angular/core';
+
+@Directive({
+  selector: 'a[href]'
```

Comment:
yes, the directive will spot all `a[href]` elements and will apply the directive

Reply:
I understand that and I think it makes sense to have this kind of consistent behavior.

What I am not sure about it what it takes to activate the component in an app. Just importing `DspActionModule` in the app's `AppModule`?

So when someone does not import `DspActionModule` the directive won't take effect?

I think I am a bit confused because normally we make selectors using the prefix `dsp`. And as long as a selector is not used, the directive does not take any effect, although the module is imported.

Reply:
If this is the case, I think it would be good to document it, e.g. in the design docs.